src/hotspot/share/utilities/powerOfTwo.hpp

```diff
@@ -123,6 +123,8 @@ inline T next_power_of_2(T value)  {
 // Find log2 value greater than this input
 template <typename T, ENABLE_IF(std::is_integral<T>::value)>
 inline T ceil_log2(T value) {
+  assert(value > 0, "Invalid value");
+  if (value == 1) return 0;
```

Comment:
Is this the same as starting the loop from `ret = 0`?

Reply:
Yeah, I think so. Pushed an update since I agree it reads better that way. Thanks!